src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -31,14 +36,25 @@ export class SearchResults extends Component {
     ),
     page: PropTypes.number.isRequired,
     perPage: PropTypes.number.isRequired,
+    showYellowRibbonEnhancements: PropTypes.bool,
     totalResults: PropTypes.number,
   };
 
-  componentDidUpdate(prevProps) {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isSearchResultsTipOpen: true,
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState) {
     const justRefreshed = prevProps.fetching && !this.props.fetching;
 
     if (justRefreshed) {
       focusElement('[data-display-results-header]');
+      if (!prevState.isSearchResultsTipOpen) {
+        this.toggleAlertBoxAppearance(true)(null);
+      }
```

Comment:
It's kinda of a bummer to have to update state in `componentDidUpdate` but offhand I don't have a better idea

Reply:
Yeah, I milled over this one for a bit and couldn't come up with another way.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/15897#discussion_r570323749 this would work 🙂

Reply:
Yeah, I have a tendency only to use redux when I need to. I tend to put state locally first before uplifting it to redux. 
Redux might be the only way here then if we're concerned about a potential infinite loop.